templates/support/index.html

```diff
@@ -297,6 +302,16 @@ <h2>Apps support</h2>
               <img src="https://res.cloudinary.com/canonical/image/fetch/f_auto,q_auto,fl_sanitize,w_16,h_16/https://assets.ubuntu.com/v1/f1a7515d-tick-darkaubergine.svg">
             </div>
           </td>
+          <td class="u-align--center">
+            <div class="lazy-loaded">
+              <img src="https://res.cloudinary.com/canonical/image/fetch/f_auto,q_auto,fl_sanitize,w_16,h_16/https://assets.ubuntu.com/v1/f1a7515d-tick-darkaubergine.svg">
```

Comment:
Every instance of this tick icon on this page needs to include alt text:
```suggestion
              <img src="https://res.cloudinary.com/canonical/image/fetch/f_auto,q_auto,fl_sanitize,w_16,h_16/https://assets.ubuntu.com/v1/f1a7515d-tick-darkaubergine.svg" alt="Included">
```